docs/getting-started.md

```diff
@@ -411,11 +409,11 @@ Here are the highlights of the template above:
 So far, we've been using `m.render` to manually redraw after we made a change to the data. However, as I mentioned before, you can enable an [auto-redrawing system](auto-redrawing.md), by initializing the `todo` component via `m.mount`.
 
 ```javascript
-//render the todo component inside the document DOM node
-m.mount(document, {controller: todo.controller, view: todo.view});
+//render the todo component inside the body DOM node
```

Comment:
Nit: maybe drop the "...inside [...]" part entirely? It just reads weird otherwise IMO.